File: src/main/java/com/jill/carRental/entity/Car.java
package com.jill.carRental.entity;

import io.micrometer.common.lang.Nullable;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
// insert into car (car_model, car_name, car_number, price, type, is_available, is_insured, insurance_type) values ('Anniversary Edition', 'Ertiga', 'MH 04 9641', 1500000.0, 'SUV'), ('Basic', 'Ritz', 'MH 03 9641', 500000.0, 'COMPACT', );


@Data
@AllArgsConstructor
@Entity
@NoArgsConstructor
public class Car {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int carID;
    private String carNumber;
    private String carName;
    private String carModel;
    private String type;
    private double price;
    private boolean isAvailable;
    private boolean isInsured;
    @Nullable
    private String insuranceType;

}

File: Frontend/src/components/Car.js
import React from 'react';

const Car = ({ car }) => {
  return (
    <div>
      <h2>{car.carName}</h2>
      <ul>
        <li>
          Car Model: {car.carModel} 
        </li>
        <li>
          Car Number: {car.carNumber} 
        </li>
        <li>
          Type: {car.type} 
        </li>
        <li>
          Available: Rs.{car.price} 
        </li>
        <li>
          Insured: Rs.{car.price} 
        </li>
        <li>
          Insured type: Rs.{car.price} 
        </li>
        <li>
          Price: Rs.{car.price} 
        </li>
      </ul>
    </div>
  );
};

export default Car;
